modular_skyrat/modules/cellguns/code/mediguns.dm

```diff
@@ -274,13 +274,13 @@
 	ammo_type = /obj/item/ammo_casing/energy/medical/utility/body_teleporter
 
 /obj/item/weaponcell/medical/utility/relocation
-	name = "Oppressive Force Relocation Medicell"
+	name = "oppressive force relocation medicell"
```

Comment:
```suggestion
	name = "Oppressive Force relocation medicell"
```
It's entirely up to you, but I think this looks better.

Reply:
agree actually.